documentation/api/definitions.md

```diff
@@ -281,6 +282,8 @@ Detailed information about an operation.
 | `operationId` | guid | Unique identifier for the operation. |
 | `createdDateTime` | datetime string | UTC DateTime string of when the operation was created. |
 | `status` | [OperationState](#operationstate) | The current status of operation. |
+| `egressProviderName` | string | The name of the egress provider that the artifact is being sent to. This will be null if the artifact is being sent directly back to the user from an HTTP request. |
```

Comment:
For all of the new functionality, let's add the `8.0+` notion so we can call out these are not applicable to the 7.0 release that just went out.